parm/mom6/MOM_input_template_100

```diff
@@ -829,6 +829,28 @@ ENERGYSAVEDAYS = 0.25           !   [days] default = 1.0
                                 ! other globally summed diagnostics.
 
 ! === module ocean_model_init ===
+ODA_INCUPD = @[ODA_INCUPD]   ! [Boolean] default = False
```

Comment:
So usually the MOM_input_template_* is a copy of what is in ufs-weather-model.  Do these updates need a corresponding test in ufs-weather-model?

Reply:
I think there is one here: https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/tests/datm_cdeps_iau_gefs but it looks like the settings are different

Reply:
It seems close enough to me @JessicaMeixner-NOAA but up to you if you think it needs an update to the RT.
@aerorahul was trying to bring the naming nomenclature closer to what the atmosphere DA is doing, which does make sense.

Reply:
That sounds like a great idea to me too.  I think it's important that we are testing in ufs-weather-model what we're actually going to be running and having consistency makes updating the model in the future easier.  I believe there was talk at one point to actually point to the MOM_template files in ufs-eather-model instead of having a second copy of these, but I'm not sure if that's still going to be a future development for g-w or not.  Either way, I just wanted to flag this as something to be aware of, but I don't think it should hold up this PR.

Reply:
In principle, we should use the templates from the ufs-weather-model for these.
If the settings are different or not templated, then they need to be templated to allow for cycling/workflow v/s model/RT settings.  It might need an update to the RT.

if we use the templates for the coupled model from the ufs-weather-model, is there someone (e.g. from the coupled modeling team) to ensure that when they are updated (new variables added/ old variables removed) in the RT, this information gets relayed to the DA and the workflow teams and/or updated in the workflow configurations?

Reply:
@aerorahul  Whenever I update the version of ufs-weather-model one of the checks is to make sure that the inputs are kept in sync.  With the HR1 PR this is true for ice and ocean except for I haven't added the CICE B-grid, as it's still mostly testing mode.     I think it's a fair statement to say that I'm frequently the person updating the model in the global-workflow and checking namelists is a part of that process.

Reply:
Thanks @JessicaMeixner-NOAA 
We can let this PR pass and I can open a follow up PR removing these files from global-workflow and using the ones from ufs-weather-model.  
Would you be able to provide a  branch of the ufs-weather-model that allows for this flexibility? 
The ufs-weather-model RT parm/ files may need an update.

Reply:
Yes. 3. is on our radar. 
We need atleast 1. for 3.  
There is a lot more work in the ufs-weather-model than in the global-workflow to achieve this.
I can create issues in the global-workflow and ufs-weather-model and assign us respectively if that is ok?

Reply:
That sounds great.  Just know this isn't anywhere near the top of my priority list at the moment.

Reply:
@JessicaMeixner-NOAA 
https://github.com/NOAA-EMC/global-workflow/issues/1312
https://github.com/ufs-community/ufs-weather-model/issues/1613
have been created